application/controllers/rest/V1/OpenApi/Schema/Question/QuestionDetail.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+/**
+ * @OA\Schema(
```

Comment:
@fosteruk Heyo, here's the PR, answer when time allows, no stress.

Seems like nothing or few things in Swagger can be generated automatically? Or?